package.json

```diff
@@ -72,15 +73,20 @@
     "grunt-contrib-clean": "^2.0.0",
     "grunt-contrib-coffee": "^2.0.0",
     "grunt-contrib-concat": "^1.0.1",
-    "grunt-contrib-jasmine": "^2.0.3",
+    "grunt-contrib-jasmine": "^2.2.0",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-shell": "^3.0.0",
+    "handlebars": "^4.7.6",
     "istanbul": "0.4.5",
     "jasmine": "3.4.0",
     "jasmine-spec-reporter": "^4.2.1",
+    "npm-force-resolutions": "0.0.3",
     "sinon": "^7.0.0",
     "sphere-coffeelint": "git://github.com/sphereio/sphere-coffeelint.git#master"
   },
+  "resolutions": {
+    "handlebars": "^4.7.6"
```

Comment:
just a question here, are we sure this works for all libraries that depend on `handlebars` below version 4?

Reply:
It is being used by istanbul and current version is 4.0.1 so migrating to 4.7.6 would not create any problem on production. What do you say ?

Reply:
Sounds great, thanks!